orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/scripts/backup_db.sh

```diff
@@ -29,6 +29,11 @@ RANGERGROUP="$5"
 LOGFILE=${6:-/var/log/}/dl_postgres_backup.log
 echo "Logs at ${LOGFILE}"
 
+if [[ -f /hadoopfs/fs1/database-cacerts/certs.pem ]]; then
```

Comment:
how about if we pass the folder /hadoopfs/fs1 down  in a pillar. In that case if somebody change the volume logic then we automatically get the right value.

Reply:
Where should we set that pillar value? Is it already available in salt through the other changes @lajosrodek has made, or will I need to set it somewhere in the Java side?

Reply:
That would certainly need changes in my logic as well once it happens. 😃

@doktoric I agree this would be a better approach, and we shall indeed do it (not only here but also in other related parts). However, I am wondering if it is the right time to fix this now. Could we possibly postpone it to 2.33 and fix all affected things altogether?

Reply:
I've added a new value under the disaster_recovery pillar.

Reply:
I'll monitor the build jobs and retry them.

Reply:
Has been solved in #9563.